plugins/weightedECDSA/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { KernelValidator } from "@zerodev/sdk/types"
+import {
+    createWeightedECDSAValidator,
+    updateConfig
+} from "./toWeigthedECDSAValidatorPlugin.js"
+
+export { createWeightedECDSAValidator, updateConfig, type KernelValidator }
```

Comment:
Why are you re-exporting the `KernelValidator` type?

Reply:
I've copied the existing ecdsa plugin and it re-exports `KernelValidator` type. Not sure but I've just followed the ecdsa plugin code.

Reply:
If we can remove it, then I'll delete it in ecdsa plugin too

Reply:
Oh hmm @SahilVasava why are we re-exporting it?